core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -443,7 +443,7 @@ var StoryEditorPage = function() {
         var selectedSkill = skillListItems.get(index);
         await waitFor.elementToBeClickable(
           selectedSkill, 'selectedSkill takes too long to be clickable.');
-        await selectedSkill.click();
+        await selectedSkill.action.click();
```

Comment:
Same as above. You can also delete the `waitFor.elementToBeClickable()` call since it is incorporated in `action.click()`

Reply:
Done.